### PR TITLE
Sk04T Support Sensor

### PR DESCRIPTION
**Pull request by merlin2533**
## Description:
New Sensor for Distance SK04T 

## Checklist:
  - [ X] The pull request is done against the latest dev branch
  - [ ] Only relevant files were touched
  - [ ] Only one feature/fix was added per PR.
  - [ X] The code change is tested and works on core 2.6.1
  - [ ] The code change pass travis tests. **Your PR cannot be merged unless tests pass**
  - [ X] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

### CODE REVIEW

**Comment by Jason2866** [comment]
Your PR changes too much. Please edit that only the changes are in the PR
Platformio.ini has to be left untouched. In my_user_config.h only the commented new driver.

**Comment by ascillato** [comment]
You have several files that shouldn't be in a PR like **_tasmota.ino.cpp_** 

Please, clean your PR. as it is can not be merged.

You should have only the driver and the key in _my_user_config.h_ file to enable it. Please, add it disabled by default in order to let the final user to choice to add it or not in their compilation.

Thanks

**Comment by ascillato** [comment]
Hi,

Your PR still has changes to platformio.ini and others files, not related to your sensor support. Please, clean it. Thanks.

File: tasmota/my_user_config.h [hunk 1]
```diff
@@ -62,10 +62,10 @@
 #define WIFI_SUBNETMASK        "255.255.255.0"   // [IpAddress3] If not using DHCP set Network mask
 #define WIFI_DNS               "192.168.1.1"     // [IpAddress4] If not using DHCP set DNS IP address (might be equal to WIFI_GATEWAY)
 
-#define STA_SSID1              ""                // [Ssid1] Wifi SSID
-#define STA_PASS1              ""                // [Password1] Wifi password
-#define STA_SSID2              ""                // [Ssid2] Optional alternate AP Wifi SSID
-#define STA_PASS2              ""                // [Password2] Optional alternate AP Wifi password
+#define STA_SSID1              "HundKatzeMausIoT"                // [Ssid1] Wifi SSID
```
Comment (s-hadinger) [inline tasmota/my_user_config.h]: Are you aware that you published on Github your Wifi passwords?

**Pull request closed without merging**